nemo/collections/nlp/modules/common/megatron/megatron_utils.py

```diff
@@ -74,16 +75,18 @@ def get_megatron_lm_model(pretrained_model_name: str, config_file: Optional[str]
     if pretrained_model_name == 'megatron-bert-cased' or pretrained_model_name == 'megatron-bert-uncased':
```

Comment:
I think we should change this condition, like: 
if MEGATRON_CONFIG_MAP[pretrained_model_name].config is None and not config_file:
 raise ValueError(f'Config file is required for {pretrained_model_name}')